test/vvenc_unit_test/vvenc_unit_test.cpp

```diff
@@ -690,40 +720,44 @@ static bool check_EqualCoeffComputer( AffineGradientSearch* ref, AffineGradientS
   for( int b6Param : { 0, 1 } )
   {
     std::ostringstream sstm_test;
-    sstm_test << "AffineGradientSearch::EqualCoeffComputer" << " b6Param=" << std::boolalpha
-              << static_cast<bool>( b6Param );
+    sstm_test << "AffineGradientSearch::EqualCoeffComputer<" << std::boolalpha << static_cast<bool>( b6Param ) << ">"
+              << " Input=" << inp_gen.input_type();
     std::cout << "Testing " << sstm_test.str() << std::endl;
 
-    for( unsigned n = 0; n < num_cases; n++ )
+    // Set height and width to powers of two >= 8.
+    for( int height : { 8, 16, 32, 64, 128 } )
```

Comment:
In VVC, affine is only allowed for width > 8 && height > 8, i.e. no size can be smaller than 16. Could you adapt?